charts/spire/README.md

```diff
@@ -7,6 +7,48 @@ A Helm chart for deploying the complete Spire stack including: spire-server, spi
 
 **Homepage:** <https://github.com/spiffe/helm-charts/tree/main/charts/spire>
 
+## Install notes
```

Comment:
Moved the install/upgrade notes to the top, as thats the most likely course of action by the user.